### ISSUE TITLE
[a11y] Combobox that cannot have items added shows Add

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1573]
_I argue this is distinct from https://github.com/specify/specify7/issues/108 and I think this needs to be addressed_

On `edge` you cannot add a pick list item- which is good behavior.

![image](https://user-images.githubusercontent.com/37256050/168158271-b3ee6d97-e848-4b25-a502-da143301de87.png)

On `a11y` the pick list gives the user the impression that they can add an item, only to throw an error and 
cause the user to lose all of their progress.

![image](https://user-images.githubusercontent.com/37256050/168158285-09f5102a-b270-4fc7-a2fa-e96428d64310.png)

Maybe we should consider https://github.com/specify/specify7/issues/1025 or revert to `edge`'s behavior
